### ISSUE TITLE
Compilation problems (on Mint).

### ISSUE THREAD

**Issue by Achaean** [beniz/seeks#30]
Hi! :-)

I\m trying to compile Seeks on Mint 17.3 x64 KDE, however ./configuration doesn't seems to complete successfully.
Testing: gtest  returns  no, although libgtest-dev is installed.
Also at Enabled plugins:
HTTP server: -> false and
XSL serializer: -> false

For HTTP I believe Apache2-dev and
for XSL libsaxon-java are what they need (please correct me, if I'm wrong. I'm not an expert on web developing). :-)
However, installing these doesn't do something.

Thanks! :-)
G.

**Comment by Quix0r** [comment]
You need to re-run configure to let it find the new headers. And seeks is long-time unmaintained (lacking new maintainer). Maybe searx is something else you want? It doesn't work the same way (no "peer-2-peer" and no personalized results). But it runs really stable here, only sometimes I get timeouts and then no results. Maybe I enabled to much engines? ;-)

**Comment by Achaean** [comment]
THANKS Quix0r! :-)

I'll look again, at configure file.
I didn't know about Searx!
Looks promising. I'll definitely, give it a try.

I was testing YaCy these days.
It can't compete the well known corporate engines of course, but (besides the fact of being open source), the strong point of these engines is that they're giving results that are virtually invisible when Googling.
PS. I don't really like, the "personalization" of the results.

Bye! :-)
A.

**Comment by Quix0r** [comment]
Personalizazion is not distributed in seeks, it remains on your node. So if you don't like it, keep the node not reachable from Internet.

I dropped YaCy some years ago, it runs only stable if you keep adding more RAM. I don't know how it is now, that they have overcome this "ever-increasing-memory-consumption" bug but maybe you can try it out and only crawl not much pages.

But that again makes no sense again, still those corporate search engines have tons of more URLs crawled and so YaCy cannot compete with them (as a user's view).

**Comment by GreenLunar** [comment]
> But that again makes no sense again, still those corporate search engines have tons of more URLs crawled and so YaCy cannot compete with them (as a user's view).

They provide popular or biased URLs; they do not provide enough of good quality references; sometimes, they add more URLs to their blacklists (censorship).

Be aware that more and more users have bookmarks databases that exceed 100 MB and even 500 MB, and most of their bookmarks entries are not related to entertainment - they do not rely on corporate search engines.

We should not and need not to attempt to compete, but rather think of serving hundreds or several thousands of users; so it is with XMPP today, even though XMPP surpasses all corporate IM networks and it appears that most people do not use it.

**Comment by Quix0r** [comment]
Yes, because most people simply don't care about it as long as they don't need to pay (with money) for it. Well, getting really OT here now.

And very well said. :)
